### PR TITLE
Storing abi with major minor version only removed patch version

### PR DESCRIPTION
**Pull request by sarvesh-ost**
Follow up PR for #85 
Storing ABI on folder name `MAJOR.MINOR` instead of `MAJOR.MINOR.PATCH`. 

Reason: patch version doesn't change ABI

### CODE REVIEW

File: tools/abi_downloader/index.ts [hunk 1]
```diff
@@ -17,10 +17,12 @@ const allowedABIs = [
 ];
 
 function writeToFile(name, abi, contractVersion): void {
+  const patchVersionIndex = contractVersion.lastIndexOf('.');
+  const majorMinorVersion = contractVersion.slice(0, patchVersionIndex);
```
Comment (deepesh-kn) [inline tools/abi_downloader/index.ts]: User may not send the patch version `0.10` instead of `0.10.0`

Comment (sarvesh-ost) [reply tools/abi_downloader/index.ts]: If correct version is not passed, npm install will fail. This code will execute only for correct version.
https://github.com/OpenST/mosaic-chains/blob/b0698f9211ddf2a780e01253a8645acc2e657f6e/tools/abi_downloader/run.sh#L10

File: tools/abi_downloader/index.ts [hunk 2]
```diff
@@ -16,11 +16,30 @@ const allowedABIs = [
...
 ];
 
-function writeToFile(name, abi, contractVersion): void {
+/**
+ * This method removes patch contractVersion from the contractVersion if any.
+ * @param contractVersion Version of contract.
+ */
+function getMajorMinorVersion(contractVersion: string): string {
+  // Check if patch version exists.
+  if (contractVersion.split('.').length > 2) {
```
Comment (deepesh-kn) [inline tools/abi_downloader/index.ts]: It should fail when the length is 1. This check can go in `tools/abi_downloader/run.sh` file.

Comment (sarvesh-ost) [reply tools/abi_downloader/index.ts]: `npm install @openst/mosaic-contracts@0` is valid version of package.

Comment (deepesh-kn) [reply tools/abi_downloader/index.ts]: True, but we should not allow it. We should have a major-minor version else we cannot differentiate between `v0.10` and `v0.11`

Comment (sarvesh-ost) [reply tools/abi_downloader/index.ts]: In that case, I will add validation for all three version `MAJOR.MINOR.PATCH` and remove this logic here.  OK?

**Pull request merged**